### PR TITLE
Cluster monitoring - Cluster status and Alert list

### PR DESCRIPTION
**Pull request by MonPote**
**Component**: ui, monitoring

**Before merge please make sure that #1142 #1155 are merged !**

**Context**: 
Solve : #1134 #1133
Blocked by : #1142 #1155
This PR have for purpose to display the first version of the cluster monitoring that :
- Display the cluster status #1134
- Display the list of alert #1133

**Summary**:
Here the list of things that has been done in this PR. The scope is a bit bigger than the 2 issues above.
- Display the cluster status
- Display the list of alert (both in this [screenshot](https://puu.sh/DxqMq/886cf3cc5e.png))
- Improve and standardize the UI ([sc1](https://puu.sh/DxGJb/fac303021c.png) and [sc2](https://puu.sh/DxGJQ/bfdbf6134d.png))
- Fixed part of the refresh issue (it will not redirect you to `/` but you still lose the path)

**Acceptance criteria**: 
To validate this PR when you test it, you should have the same look & feel of `screenshot`, `sc1` and `sc2` above.

If you want to test this PR before #1142 #1155 merged please follow this process in order to install prometheus properly and setup the ui locally.
Prerequisite : 
- A cluster up and running with a least a node up.

Then you should : 
- ssh into your `boostrap` node and clone this repo `https://github.com/coreos/prometheus-operator/tree/v0.24.0`
- go to the folder `/contrib/kube-prometheus/manifests/` and run : 
```
for file in *; do
    kubectl apply -f $file
done
```
- after a few sec you should run `kubectl get svc -n monitoring` see the service `prometheus-k8s` up and running 
- you can now run `kubectl -n monitoring expose service prometheus-k8s --type=NodePort --name=prometheus-k8s-nodeport`
- you need to find the ip and port where the ui will hit prometheus (IP=`ip a` and get `eth1` ip; PORT=`kubectl get svc -n monitoring` and get `prometheus-k8s-nodeport` port.)
- change `url_prometheus` inside `metalk8s/ui/public/config.json` and replace it by `http://{IP}:{PORT}` (replace `IP` and `PORT` by the value found above)
- you can now run the UI with `npm install && npm run start` in `metalk8s/ui/` folder and test this PR.

**Known issues** We might want to create issue to track it.
- When you hover the row of a table, the hover hides the line and it's not very aesthetics
- The header of a table is not totally align with his body (on the right).
- When you refresh a page, you stay on the same page (it's correct) but you lose the url path.
- Since the design (UI) has not been reviewed, it might subject to debate and improvement, feel free to give suggestion.
- **I have not the opportunity to test it correctly with the PR #1142 #1155 so this PR might break after the merge !**

**Before merge please make sure that #1142 #1155 are merged !**

### CODE REVIEW

File: ui/src/containers/ClusterStatus.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+`;
+
+const CusterStatus = props => {
+  useEffect(() => {
+    props.fetchAlerts();
+  }, []);
+
+  const columns = [
+    {
+      label: 'Name',
```
Comment (carlito-scality) [inline ui/src/containers/ClusterStatus.js]: i18n please

File: ui/src/containers/ClusterStatus.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    {
+      label: 'activeAt',
+      dataKey: 'activeAt',
+      width: 230
+    }
+  ];
+
+  const data = props.alerts.map(alert => {
+    const activeAtMoment = moment(alert.activeAt);
+    const alertActiveAt = activeAtMoment.format('YYYY-MM-DD HH:mm:ss Z');
```
Comment (carlito-scality) [inline ui/src/containers/ClusterStatus.js]: you should use  FormattedDate, FormattedTime of 'react-intl' instead of moment

**Comment by MonPote** [comment]
@bert-e approve

**Comment by gdemonet** [comment]
@bert-e after_pull_request=1142

**Comment by gdemonet** [comment]
@bert-e after_pull_request=1155

**Comment by carlito-scality** [comment]
You should add unit tests and some minor changes but the code LGTM

File: ui/src/index.js [hunk 3]
```diff
@@ -24,7 +23,7 @@ sagaMiddleware.run(sagas);
 
 ReactDOM.render(
   <Provider store={store}>
-    <Router history={history}>
```
Comment (carlito-scality) [inline ui/src/index.js]: it is not the root cause. See https://github.com/scality/metalk8s/pull/1140/commits/426692694934ef805dceb56a0206670e219ebd07

File: ui/src/containers/ClusterMonitoring.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+const ClusterStatusTitleContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const ClusterStatusValue = styled.span`
+  margin-left: ${padding.small};
+  font-weight: bold;
+  color: ${props => (props.isUp ? brand.success : brand.danger)};
```
Comment (carlito-scality) [inline ui/src/containers/ClusterMonitoring.js]: you should merge rebrand theme and default theme

File: ui/src/containers/ClusterMonitoring.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+      name: alert.labels.alertname,
+      severity: alert.labels.severity,
+      message: alert.annotations.message,
+      activeAt: alert.activeAt
+    };
+  });
+
+  const sortedAlerts = sortAlerts(alerts, sortBy, sortDirection);
+
+  const isUp = props.clusterStatus && props.clusterStatus.length === 0;
```
Comment (carlito-scality) [inline ui/src/containers/ClusterMonitoring.js]: it should be ```const isUp = props.clusterStatus && props.clusterStatus.length; ```?

**Comment by jbertran** [comment]
Rebased on top of #1155, works perfectly despite the monitoring services being on fire.

**Review by jbertran — APPROVED** [review]

**Comment by MonPote** [comment]
@bert-e approve

**Comment by MonPote** [comment]
@bert-e reset

**Pull request merged**